File: LTHAM/151/151.cpp
#include <iostream>
using namespace std;

int ktDang2m(int);

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	if (ktDang2m(n) == 1)
		cout << "La dang 2^m" << endl;
	else
		cout << "Khong la dang 2^m" << endl;
	return 0;
}

int ktDang2m(int n)
{
	int flag = 1;
	int t = n;
	while (t > 1)
	{
		int du = t % 2;
		if (du != 0)
			flag = 0;
		t = t / 2;
	}
	return flag;
}

File: CTDK_Python/151/_151.py
n = int(input("Nhap n: "))

flag = 1
t = n

while t > 1:
    du = t % 2
    if (du != 0):
        flag = 0
    t = t // 2

if flag == 1:
    print("Co dang 2^m")
else:
    print("Khong co dang 2^m")




